docs/05-plugin-development.md

```diff
@@ -187,6 +187,23 @@ Kind: `async, parallel`
 
 Called initially each time `bundle.generate()` or `bundle.write()` is called. To get notified when generation has completed, use the `generateBundle` and `renderError` hooks.
 
+#### `augmentChunkHash`
```

Comment:
Thanks for adding documentation! It would be nice to stick with the alphabetical ordering of the hooks and move this up.